src/openassetio-python/managerApi/ManagerInterfaceBinding.cpp

```diff
@@ -97,7 +101,9 @@ void registerManagerInterface(const py::module& mod) {
       .def("createState", &ManagerInterface::createState, py::arg("hostSession").none(false))
       .def("createChildState", &ManagerInterface::createChildState,
```

Comment:
[question] Does `parentState` need the same treatment?

Reply:
Good point.  We can't be certain that implementations wouldn't keep hold of a reference.

Reply:
Added as part of 4e4c1fed